airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -858,15 +855,9 @@ private void runSyncAndVerifyStateOutput(JsonNode config, List<AirbyteMessage> m
 
     destination.close();
 
-    // skip if basic normalization is not configured to run (either not set or false).
-    if (!config.hasNonNull(WorkerConstants.BASIC_NORMALIZATION_KEY) || !config.get(WorkerConstants.BASIC_NORMALIZATION_KEY).asBoolean()) {
```

Comment:
@ChristopheDuong don't we still need some sort of check here to make sure that the destination actually _can_ support basic normalization?

Reply:
@ChristopheDuong  Don't we need a field in [connector specification](https://github.com/airbytehq/airbyte/blob/master/airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml#L233-L243) for the destination that signal whether or not it can support basic normalization or not?

Right now we were using the presence of the `basic_normalization` field in the config to do things: 1. determine if the destination supported basic normalization and 2. know if the it was turned on and off. What i'm seeing so far is that we are solving 2 in the standard sync (which makes sense), but I'm not understanding where 1 happens. It's definitely not safe to assume all destinations support basic normalization. Is that going to happen somewhere else?

Reply:
Yes, we should probably add a `supportsDbt` field as we have with `supportsIncremental` at the top of the destination spec.json

I renamed it as `supportsDbt` instead of `supportsNormalization` because it's also a check we would need for custom dbt transformation too.

Reply:
After chatting with @cgardens , it might be best to actually split in two booleans:

- `supportsDbt`: for all destinations that dbt supports (includes MySQL, Mssql, oracle)
- `supportsNormalization`: for all destinations that used to have `basic_normalization = true` (MySQL, MSSQL, Oracle are not implemented yet)